examples/fuse_examples/imaging/classification/knight/baseline/config.yaml

```diff
@@ -1,21 +1,25 @@
 experiment_num : 0
-task_num : task_2 # task_1 or task_2
+task_num : task_1 # task_1 or task_2
 num_gpus : 1
 use_data : {"imaging": True, "clinical": True}
-batch_size : 2
-resize_to : [70, 256, 256]
-num_epochs : 150
-learning_rate : 0.0001
+batch_size : 6
```

Comment:
Are you using a much bigger GPU?
If yes, can add in comment here that for smaller GPUs, the batch size should be smaller?